backend/FwLite/MiniLcm/Normalization/MiniLcmApiStringNormalizationWrapper.cs

```diff
@@ -0,0 +1,46 @@
+using System.Text;
+using MiniLcm;
+using MiniLcm.Models;
+using MiniLcm.SyncHelpers;
+using MiniLcm.Wrappers;
+
+namespace MiniLcm.Validators;
+
+public class MiniLcmApiStringNormalizationWrapperFactory(NormalizationForm normalization = NormalizationForm.FormD) : IMiniLcmWrapperFactory
+{
+    public IMiniLcmApi Create(IMiniLcmApi api, IProjectIdentifier _unused) => Create(api);
+
+    public IMiniLcmApi Create(IMiniLcmApi api)
+    {
+        return new MiniLcmApiStringNormalizationWrapper(api, normalization);
+    }
+}
+
+public partial class MiniLcmApiStringNormalizationWrapper(
```

Comment:
let's test this wrapper. Should be pretty easy to mock with the moq library in MiniLcm tests.